app/code/Magento/Shipping/Model/Simplexml/Element.php

```diff
@@ -32,9 +32,9 @@ public function addAttribute($name, $value = null, $namespace = null)
      * @param string $name The name of the child element to add.
      * @param string $value If specified, the value of the child element.
      * @param string $namespace If specified, the namespace to which the child element belongs.
-     * @return \Magento\Shipping\Model\Simplexml\Element
+     * @return \Magento\Shipping\Model\Simplexml\Element|null
      */
-    public function addChild($name, $value = null, $namespace = null)
+    public function addChild($name, $value = null, $namespace = null): ?self
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function addChild($name, $value = null, $namespace = null)
```

Reply:
Hello @ihor-sviziev
This is really true.
This is big BIC IMHO.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.